src/jit/lsra.cpp

```diff
@@ -1914,6 +1914,8 @@ bool LinearScan::isRegCandidate(LclVarDsc* varDsc)
 
 void LinearScan::identifyCandidates()
 {
+    VarSetOps::AssignNoCopy(compiler, resolutionCandidateVars, VarSetOps::MakeEmpty(compiler));
```

Comment:
Any reason this cannot be done in the class constructor or do you prefer having these here?